### ISSUE TITLE
Bug: Drawer not closing on click outside of drawer

### ISSUE THREAD

**Issue by Miaplacidus** [hicommonwealth/commonwealth#6432]
To reproduce: navigate to a thread view page and click on 'View upvotes'.  Once the drawer opens, click anywhere outside of it.
The drawer should close, but it doesn't.

It's a mystery. Fetch your deerstalker hat!

@jnaviask @ForestMars

**Comment by dillchen** [comment]
is this just potentially a simple Clickaway Listener Issue?

**Comment by masvelio** [comment]
@dillchen nope. Notice it works fine in comments. I described the root cause [here](https://github.com/hicommonwealth/commonwealth/pull/6361/files#r1466314263) but tldr, we prevent event to fire up to prevent moving user from discussions page to thread page when they click action button. The same prevention catches "click-away" from the drawer.

**Comment by dillchen** [comment]
I see! ty for that @masvelio didn't see comment. looking forward to getting this in!
